upb/msg.c

```diff
@@ -361,13 +368,17 @@ void upb_msgfactory_free(upb_msgfactory *f) {
   upb_gfree(f);
 }
 
+const upb_symtab *upb_msgfactory_symtab(const upb_msgfactory *f) {
+  return f->symtab;
+}
+
 /* Requires:
  * - m is in upb_msgfactory_symtab(f)
```

Comment:
update comment, maybe add todo about making it threadsafe and removing non-const method?